lib/ansible/modules/packaging/os/pulp_repo.py

```diff
@@ -561,9 +563,15 @@ def main():
     add_export_distributor = module.params['add_export_distributor']
     feed = module.params['feed']
     generate_sqlite = module.params['generate_sqlite']
-    importer_ssl_ca_cert = module.params['ca_cert']
-    importer_ssl_client_cert = module.params['client_cert']
-    importer_ssl_client_key = module.params['client_key']
+    importer_ssl_ca_cert = module.params['feed_ca_cert']
+    importer_ssl_client_cert = module.params['feed_client_cert']
+    if importer_ssl_client_cert is None and module.params['client_cert'] is not None:
+        importer_ssl_client_cert = module.params['client_cert']
+        module.deprecate("In Ansible 2.10 `feed_client_cert` option was added. Until 2.14 the default value will come from client_cert option", version="2.14")
```

Comment:
How about:
```suggestion
        module.deprecate("To specify client certificates to be used with the repo to sync, and not for communication with pulp.io, use the new options `feed_client_cert` and `feed_client_key` (available since Ansible 2.10). Until Ansible 2.14, the default value for `feed_client_cert` will be taken from `client_cert` if only the latter is specified", version="2.14")
```
(And something similar below.)